src/random.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const random = () => {
+  const generatedRandom = Math.random().toString().slice(2);
```

Comment:
The random number generation method may not provide a 4-digit number with unique digits. Consider using a different approach.